controlplane/kubeadm/internal/cluster.go

```diff
@@ -251,6 +266,37 @@ func (c *cluster) controlPlaneIsHealthy(ctx context.Context) (healthCheckResult,
 	return response, nil
 }
 
+func (c *cluster) removeMemberForNode(ctx context.Context, nodeName string) error {
+	tlsConfig, err := c.generateEtcdTLSClientBundle()
+	if err != nil {
+		return err
+	}
+
+	// Create the etcd client for the etcd Pod scheduled on the Node
+	etcdClient, err := c.getEtcdClientForNode(nodeName, tlsConfig)
```

Comment:
Should the client be made for a node which isn't being deleted?

Reply:
Let's handle that as a followup.

Reply:
Does this need to be tracked as an issue or are we good here?

Reply:
I'll create an issue

Reply:
#2393